lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -673,6 +673,8 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
 
         add_3ds_info(parameters, options[:three_d_secure])
 
+        parameters[:sca_exemption] = options[:three_ds_exemption_type] if options[:three_ds_exemption_type]
```

Comment:
There will likely be a set amount of standard exemption types across gateways. What if we standardize them inside the `options[:three_d_secure]` [hash](https://github.com/activemerchant/active_merchant/wiki/Standardized-3DS-Fields) and then map them to the gateway's specific values kind of like `STANDARD_ERROR_CODE`
```
THREE_DS_EXEMPTIONS = {
  'low_value': 'low_value',
  'another_exemption': 'exemption_another'
}
```

Reply:
Looking around AM quite a few gateways look for something like `options[:sca_exemption]` so maybe we standardize on that field?

Reply:
`:sca_exemption` is what we map for Adyen & Barclaycard and for Redsys is what the field the customer sends us. `three_ds_exemption_type` is the field that customer sends us but we map it on our side to be specific to the gateway.

Reply:
the more you know!